Library/Homebrew/cli/parser.rb

```diff
@@ -167,6 +167,11 @@ def initialize(cmd = nil, &block)
           @command_name = T.let(T.must(cmd_location.label).chomp("_args").tr("_", "-"), String)
           @is_dev_cmd = T.let(T.must(cmd_location.absolute_path).start_with?(Commands::HOMEBREW_DEV_CMD_PATH),
                               T::Boolean)
+          odeprecated(
+            "`brew #{@command_name}'. This command needs to be refactored, as it is written in a style that",
+            "inheritance from `Homebrew::AbstractCommand' ( see https://docs.brew.sh/External-Commands )",
+            disable_for_developers: false,
+          )
```

Comment:
```suggestion
          # odeprecated(
          #   "`brew #{@command_name}'. This command needs to be refactored, as it is written in a style that",
          #   "inheritance from `Homebrew::AbstractCommand' ( see https://docs.brew.sh/External-Commands )",
          #   disable_for_developers: false,
          # )
```

as @Bo98 mentioned.

Not sure how I missed this first time, good catch @Bo98 😓